src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -181,6 +183,34 @@ private static NuGetVersion SpecifySnapshot(string version, string snapshotValue
             return new NuGetVersion(version);
         }
 
+        private static PackOptions GetPackOptions(PackageSpec packageSpec, JObject rawPackageSpec)
+        {
+            var rawPackOptions = rawPackageSpec.Value<JToken>("packOptions") as JObject;
+            if (rawPackOptions == null)
+            {
+                return new PackOptions();
+            }
+
+            var rawPackageType = rawPackOptions["packageType"];
```

Comment:
Does this need to support an array also? I think we decided there was going to be a common format for all properties where they could be either a string, an array, and possibly a third type.